src/CMakeLists.txt

```diff
@@ -1,11 +1,18 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
We are building this on some really ancient systems. I would have to check, but I would dare to bet that CMake 3.21 isn't available there yet.

I do like the change in principle, I'm just not sure how to best proceed.

Reply:
I set it this version because according to documentation, this is when PROJECT_IS_TOP_LEVEL was introduced.

Reply:
I changed the condition so that the version increase should no longer be necessary.